### PR TITLE
add a "test" to run the different tools from within VisualStudio

### PR DESCRIPTION
**Pull request by monsieurleberre**
This allows the user to run the different tools as tests and look at the output, rather than having to change the Main in Program.cs and keep the console opened.
If really someone prefers the console, they can just call the skipped tests from Program.cs

I will include FederatedPeg network instead of Apex in there as soon as https://github.com/stratisproject/FederatedSidechains/pull/182
gets merged

### CODE REVIEW

**Comment by bokobza** [comment]
not sure this is a good idea. Ultimatley we want to package this somehow to make it easy for Federation members to run it and generate public keys, don't we?

**Comment by monsieurleberre** [comment]
It still does what it did before right ? This is just easier for people who don't have or don't want LinqPad to run code and get the info that they need to create a federation.

File: src/FedKeyPairGen/GenesisMiner.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    public class GenesisMiner
+    {
+        private readonly ITestOutputHelper output;
+
+        public GenesisMiner(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        //[Fact]
```
Comment (bokobza) [inline src/FedKeyPairGen/GenesisMiner.cs]: I don't really understand what marking this as a fact does?

Comment (bokobza) [reply src/FedKeyPairGen/GenesisMiner.cs]: Ah so that you can run these methods one by one?

Comment (monsieurleberre) [reply src/FedKeyPairGen/GenesisMiner.cs]: It means you can run those functions from within visual studio and get the output from the test in the test output. Same as you were doing locally using LinqPad, but this way you don't need to use LinqPad or change Main() to run the functions. In the current version, those functions sit here unused, and someone is expected to change code or copy paste them in LinqPad to run them I think.

Comment (monsieurleberre) [reply src/FedKeyPairGen/GenesisMiner.cs]: image

Comment (monsieurleberre) [reply src/FedKeyPairGen/GenesisMiner.cs]: if someone prefers console or LinqPad really up to them, there is nothing that worked before that doesn't anymore

Comment (bokobza) [reply src/FedKeyPairGen/GenesisMiner.cs]: we want to make this a little console app. This way of using tests to run this thing is weird. It should be run once for each network so not something we should run all the time.

Comment (bokobza) [reply src/FedKeyPairGen/GenesisMiner.cs]: Maybe it's time for someone to actually create this nice little app so we can package it and be done with it.

Comment (bokobza) [reply src/FedKeyPairGen/GenesisMiner.cs]: ok then, let's put this in like this.
As long as this goes before the release and we have a little console app at the end.
I can package it with Warp.

Comment (monsieurleberre) [reply src/FedKeyPairGen/GenesisMiner.cs]: cool 👍

**Pull request merged**